CMake/packages/Findnmodl.cmake

```diff
@@ -35,7 +14,7 @@
 # Using nmodl:
 #
 # ::
-#   set(NMODL_ROOT "" CACHE PATH "Path nmodl source-to-source compiler root")
+#   set(CORENRN_NMODL_ROOT "" CACHE PATH "Path nmodl source-to-source compiler root")
```

Comment:
In this case maybe `CORENRN_` prefix makes sense to distinguish the new `NMODL` from the old implementations :+1: